internal/pkg/flink/internal/store/store.go

```diff
@@ -49,6 +50,22 @@ func (s *Store) ProcessLocalStatement(statement string) (*types.ProcessedStateme
 	}
 }
 
+func (s *Store) persistUserProperties() {
+	if s.appOptions.GetContext() != nil {
+		if err := s.appOptions.Context.SetCurrentFlinkCatalog(s.Properties.Get(config.ConfigKeyCatalog)); err != nil {
+			log.CliLogger.Errorf("error setting current flink catalog: %v", err)
```

Comment:
just a suggestion - if the users sees this 10 times after trying to execute a reset command with another key, he'll ask himself "I'm not trying set the flink catalog. Nitpick tho

```suggestion
			log.CliLogger.Errorf("error persisting current flink catalog: %v", err)
```

Reply:
True!